src/wast-parser.cc

```diff
@@ -1335,6 +1349,19 @@ Result WastParser::ParseModuleFieldList(Module* module) {
       CHECK_RESULT(Synchronize(IsModuleField));
     }
   }
+
+  Result result = Result::Ok;
```

Comment:
Maybe add a comment to the effect of "Module parsing is now completely.  Type names can now be resolved"